src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/MacProxy.cs

```diff
@@ -65,20 +66,22 @@ public Uri ExecuteProxyAutoConfiguration(SafeCreateHandle cfurl, CFProxy proxy)
                 CFRunLoopStop(runLoop);
             };
 
-            var clientContext = new CFStreamClientContext();
-            var loopSource =
+            CFStreamClientContext clientContext = new CFStreamClientContext();
```

Comment:
nit: this can be 'var'.  In general, we use 'var' sparingly.  But it's ok to use when the type is obvious like a 'new' statement or a cast.